python/doc/theory/reliability_sensitivity/sensitivity_sobol_from_pce.rst

```diff
@@ -0,0 +1,253 @@
+|                                     | Total                | :math:`\alpha_i > 0`                                                                                                     |
++-------------------------------------+----------------------+--------------------------------------------------------------------------------------------------------------------------+
+| Interaction of a group :math:`\bdu` | First order          | :math:`\alpha_i > 0 \textrm{ if } i \in \bdu, \quad \alpha_i = 0 \textrm{ if } i \not \in \bdu`                          |
++-------------------------------------+----------------------+--------------------------------------------------------------------------------------------------------------------------+
+|                                     | Total interaction    | :math:`\alpha_i >0 \textrm{ if } i \in \bdu`                                                                             |
++-------------------------------------+----------------------+--------------------------------------------------------------------------------------------------------------------------+
+| Group (closed) :math:`\bdu`         | First order (closed) | :math:`\alpha_i = 0 \textrm{ if } i \not\in \bdu, \quad i = 1, \ldots, n_X`                                              |
++-------------------------------------+----------------------+--------------------------------------------------------------------------------------------------------------------------+
+|                                     | Total                | :math:`\exists i\in\{1,\ldots, n_X\} \quad \textrm{s.t.} \quad i \in \bdu \textrm{ and } \alpha_i > 0`                   |
++-------------------------------------+----------------------+--------------------------------------------------------------------------------------------------------------------------+
+
+**Table 1.** Multi-indices involved in the first order and total Sobol' indices of a single variable :math:`i` or a group :math:`\bdu`.
+
+.. topic:: API:
+
+    - See :class:`~openturns.SobolIndicesAlgorithm` for indices based on sampling
+    - See :class:`~openturns.FunctionalChaosSobolIndices` for indices based on chaos expansion
+
+.. topic:: Examples:
+
+    - See :doc:`/auto_reliability_sensitivity/sensitivity_analysis/plot_sensitivity_sobol`
+    - See :doc:`/auto_reliability_sensitivity/sensitivity_analysis/plot_functional_chaos_sensitivity`
+
+.. topic:: References:
+
+    - Sudret, B. (2006). Global sensitivity analysis using polynomial chaos expansions. In. Proceedings of the 5th International Conference on Computational Stochastic Mechanics (CSM5), Rhodos (2006)
+    - Sudret, B. (2008). Global sensitivity analysis using polynomial chaos expansions. *Reliability engineering & system safety*, *93* (7), 964-979.
+    - Knio, O. M., & Le Maitre, O. P. (2006). Uncertainty propagation in CFD using polynomial chaos decomposition. *Fluid dynamics research*, *38* (9), 616.
```

Comment:
better move this to the bibliography as its cited in the other doc too

Reply:
Ok. Fixed at 3af5ad46b